JavaCryptographicArchitecture/src/SecretKeyFactory.crysl

```diff
@@ -20,13 +20,10 @@ ORDER
     Gets, Gens
 
 CONSTRAINTS
-	keyFactoryAlgorithm in {"PBKDF2WithHmacSHA512", "PBKDF2WithHmacSHA384", "PBKDF2WithHmacSHA256", "PBKDF2WithHmacSHA224", "PBEWithHmacSHA512AndAES_128","PBEWithHmacSHA384AndAES_128", "PBEWithHmacSHA384AndAES_128", 
-				  "PBEWithHmacSHA224AndAES_128", "PBEWithHmacSHA256AndAES_128","PBEWithHmacSHA224AndAES_256", "PBEWithHmacSHA256AndAES_256", "PBEWithHmacSHA384AndAES_256", "PBEWithHmacSHA512AndAES_256"};
```

Comment:
Please keep this for now. Enri has opened an issue about this and I had a brief look at this and noticed we need this to stay as it is.

Reply:
ok I will change it back

Reply:
https://github.com/CROSSINGTUD/Crypto-API-Rules/pull/65/commits/95ad6f030088be7b150ea7fbd25bb257dd173566